src/plugins/data/public/antlr/shared/utils.test.ts

```diff
@@ -6,54 +6,6 @@
 import { of } from 'rxjs';
 import { getRawSuggestionData$, fetchData, fetchTableSchemas, fetchColumnValues } from './utils';
 
-describe('getRawSuggestionData$', () => {
```

Comment:
Why are we deleting tests?

Reply:
this PR is not using observable, the tests are failing as these are testing against observable, changes are coming